workers.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+if ! which python3 >/dev/null; then
+  echo "Python 3 has not been configured. Make sure to install it and add it to your PATH."
+  exit 1
+fi
```

Comment:
It only checks for Linux / MacOS
May be this is the reason I'm failing on Windows.